### PR TITLE
fix(background-media): fix low-contrast focus state

### LINKED ISSUE #8995: [Background Media]: QA: Focus is not visible on 'Play' button
**Engineering info:**  
- Innovation Team engineer: @andy-blum 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2490
- DPO consulting engineer: N/A

-----

### Description

Page Element:
Play button

Issue Description:
Keyboard focus is not visible clearly on the Play button

Expected:
Keyboard focus should be visible clearly on the button.

User Impact:
When focus indicator for the page element is not clear, the user would have difficulty in identifying the currently focused element. this impacts Keyboard only users and people who with low vision

### Component(s) impacted

Background Media

### Browser

Firefox

### Carbon for IBM.com version

v1.35.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react

### Steps to reproduce the issue (if applicable)

_No response_

### Release date (if applicable)

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

### PR DESCRIPTION
**Pull request by andy-blum**
### Related Ticket(s)

Fixes #8995 

### Description

Updates the focus styles on the play/pause button for the background-media button

### Changelog

**Changed**

- Stylefix for accessible focus states

### CODE REVIEW

**Review by RichKummer — APPROVED** [review]
LGTM @andy-blum ! Thank you for this

File: packages/web-components/src/components/background-media/background-media.scss [hunk 1]
```diff
@@ -21,6 +21,13 @@
   @include tooltip--placement('definition', 'top', 'end');
 
   position: absolute;
+  border: 2px solid transparent;
```
Comment (proeung) [inline packages/web-components/src/components/background-media/background-media.scss:24]: A bit of nitpicking, but let's use the `$spacing-01` token which renders as `2px` or `0.125rem`.

```suggestion
  border: $spacing-01 solid transparent;
```

**Review by proeung — COMMENTED** [review]
Looks good! I checked the WC React Wrapper preview and the focus styling looks good as well. https://ibmdotcom-web-components-react.s3-web.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/9035/index.html?path=/story/components-background-media--with-video

**Review by kennylam — APPROVED** [review]
Thanks @andy-blum!

**Comment by proeung** [comment]
@kennylam lol duh! yeah, I failed to adjust the height of the addon section. All is good here and no further action is needed.

**Review by proeung — APPROVED** [review]

**Pull request merged**